app/views/defence_requests/show_draft.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Check the request</h1>
+
+<p>Check the details below to make sure they are correct. Update any incorrect details, then submit the request.</p>
+
+<section class="detainee-details">
+  <h2><%= not_given_formatter(@defence_request, :detainee_name, "name_not_given") %></h2>
+
+  <%= render("defence_requests/show_partials/detainee_details") %>
+
+  <%= link_to("Change this", edit_defence_request_path(@defence_request, part: "detainee_details"), class: "edit-link") %>
+</section>
+
+
+<section class="detention-details">
+  <h2 class="line-above">Detention Details</h2>
+
+  <%= render("defence_requests/show_partials/detention_details") %>
+
+  <%= link_to("Change this", edit_defence_request_path(@defence_request, part: "detention_details"), class: "edit-link") %>
+</section>
+
+<div class="draft-actions">
```

Comment:
Ok that's cool. 
We can do this in the scope of a separate story as that is a larger chunk of work